apps/build-onchain-apps/src/onchain/hooks/contracts.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Btw, we deprecated the `onchain` folder, so you can just move this into `hooks` directly.

Reply:
So in ` apps/build-onchain-apps/src/hooks/contracts.ts`